### PR TITLE
Merge request for changes to robot driver, teleop code and interface panels

### PR DESCRIPTION
**Pull request by futureneer**

### CODE REVIEW

File: ur_description/urdf/ur5.urdf.xacro [hunk 1]
```diff
@@ -56,7 +56,8 @@ center_of_mass = [ [0, -0.02561, 0.00193], [0.2125, 0, 0.11336], [0.11993, 0.0,
       <geometry>
 		    <mesh filename="package://ur_description/meshes/ur5/visual/Base.dae" />
       </geometry>
-      <origin xyz="0.0 0.0 0.003" rpy="0.0 0.0 ${3.0 * pi / 4.0}" />
+      <!-- <origin xyz="0.0 0.0 0.003" rpy="0.0 0.0 ${3.0 * pi / 4.0}" /> -->
```
Comment (jbohren) [inline ur_description/urdf/ur5.urdf.xacro:59]: Why is this commented out instead of being deleted?

File: ur_description/urdf/ur5.urdf.xacro [hunk 2]
```diff
@@ -73,7 +74,8 @@ center_of_mass = [ [0, -0.02561, 0.00193], [0.2125, 0, 0.11336], [0.11993, 0.0,
   <joint name="${prefix}shoulder_pan_joint" type="revolute">
     <parent link="${prefix}base_link" />
     <child link = "${prefix}shoulder_link" />
-    <origin xyz="0.0 0.0 ${shoulder_height}" rpy="0.0 0.0 0.0" />
+    <!-- <origin xyz="0.0 0.0 ${shoulder_height}" rpy="0.0 0.0 0.0" /> -->
```
Comment (jbohren) [inline ur_description/urdf/ur5.urdf.xacro:77]: See above.

**Comment by jbohren** [comment]
Can you just write up a summary of all of these changes in the PR description? There's a lot here.

File: ur_driver/srv/freedrive.srv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+bool active
+---
+string ack
```
Comment (jbohren) [inline ur_driver/srv/freedrive.srv:3]: This is a duplicate of free_drive.srv

**Comment by zchen24** [comment]
Pull Request Description?

File: ur_driver/srv/gettcppose.srv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+string req
+---
+Pose current
```
Comment (jbohren) [inline ur_driver/srv/gettcppose.srv:3]: Duplicate of get_tcp_pose

**Comment by jbohren** [comment]
Also can you make the message and service names conform to standard CamelCase naming conventions?

http://wiki.ros.org/ROS/Patterns/Conventions#Topics_.2BAC8_services

File: ur_driver/src/ur_driver/scratch.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+v_out = list(T.p)+[a*rot[0],a*rot[1],a*rot[2]]
+print v
+print v_out
+p_out = PyKDL.Vector(T.p)
+
+
+
+
+
+
```
Comment (jbohren) [inline ur_driver/src/ur_driver/scratch.py:27]: This looks like something that shouldn't be committed.

**Comment by jbohren** [comment]
Reverted in #2, can you re-open the PR for review?

**Pull request merged**